src/main/java/com/mipt/hsse/hssetechbackend/rent/customExceptions/RentProcessingException.java

```diff
@@ -0,0 +1,14 @@
+package com.mipt.hsse.hssetechbackend.rent.customExceptions;
```

Comment:
Вообще все пэкэджи пишутся в нижнем регистре. Вот цитата с оракла:
> Naming Conventions
> Package names are written in all lower case to avoid conflict with the names of classes or interfaces.

PS: это касается абсолютно всех файлов